install_files/ansible-base/roles/tails-config/tasks/install_shell_extension.yml

```diff
@@ -0,0 +1,123 @@
+  changed_when: false
+  register: mon_server_lookup_result
+  when: site_specific_file.stat.exists == true
+
+- name: Create the SecureDrop GNOME Shell Extension directories
+  file:
+    state: directory
+    path: "{{ item }}"
+  with_items: "{{ tails_config_extension_directories }}"
+
+- name: Set normal user ownership on subset of directories.
+  become: yes
+  file:
+    state: directory
+    path: "{{ item }}"
+    owner: amnesia
+    group: amnesia
+  # Only set normal user ownership for files in ~amnesia.
+  when: item.startswith(tails_config_amnesia_home)
+  with_items: "{{ tails_config_extension_directories }}"
+
+- name: Copy the extension metadata to the extension directory in Persistent Storage
+  become: yes
+  copy:
+    src: securedrop@securedrop.freedom.press/metadata.json
+    dest: "{{ tails_config_live_dotfiles }}/.local/share/gnome-shell/extensions/securedrop@securedrop.freedom.press/"
+    owner: amnesia
+    group: amnesia
+
+- name: Copy the extension CSS to the extension directory in Persistent Storage
+  become: yes
+  copy:
+    src: securedrop@securedrop.freedom.press/stylesheet.css
+    dest: "{{ tails_config_live_dotfiles }}/.local/share/gnome-shell/extensions/securedrop@securedrop.freedom.press/"
+    owner: amnesia
+    group: amnesia
+
+- name: Copy the symbolic icon used for the shell extension in Persistent Storage
+  become: yes
+  copy:
+    src: securedrop-symbolic.png
+    dest: "{{ tails_config_live_dotfiles }}/.local/share/gnome-shell/extensions/securedrop@securedrop.freedom.press/icons/"
+    owner: amnesia
+    group: amnesia
+
+- name: Set the right variable for source
+  set_fact:
+    source_iface: "{{ sourcev3_interface_lookup_result }}"
+
+- name: Set the right variable for journalist
+  set_fact:
+    journalist_iface: "{{ journalistv3_interface_lookup_result }}"
+
+- name: Set the right variable for app server hostname
+  set_fact:
+    app_hostname: "{{ app_server_lookup_result }}"
+
+- name: Set the right variable for app server hostname
+  set_fact:
+    mon_hostname: "{{ mon_server_lookup_result }}"
+
+- name: Assemble interface information for extension
+  set_fact:
+    _securedrop_extension_info:
+      - src: extension.js.in
+        filename: extension.js
+        source_interface_address: "{{ source_iface.stdout }}"
+        journalist_interface_address: "{{ journalist_iface.stdout }}"
+        app_hostname: "{{ app_hostname.stdout }}"
+        mon_hostname: "{{ mon_hostname.stdout }}"
+
+- name: Create SecureDrop extension
+  become: yes
+  template:
+    src: "{{ item.0.src }}"
+    dest: "{{ item.1 }}/{{ item.0.filename }}"
+    owner: amnesia
+    group: amnesia
+    mode: "0700"
+  with_nested:
+    - "{{ _securedrop_extension_info }}"
+    - "{{ tails_config_extension_directories }}"
+
+- name: Add extension translations in Persistent Storage
```

Comment:
On a fresh install, `./securedrop-admin --force tailsconfig` is failing for me here as follows:
```
TASK [tails-config : Add extension translations in Persistent Storage] *********
fatal: [localhost]: FAILED! => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --archive --out-format=<<CHANGED>>%i %n%L /home/amnesia/Persistent/securedrop/install_files/ansible-base/roles/tails-config/templates/locale/ /live/persistence/TailsData_unlocked/dotfiles/.local/share/gnome-shell/extensions/securedrop@securedrop.freedom.press/locale/", "msg": "rsync: [sender] change_dir \"/home/amnesia/Persistent/securedrop/install_files/ansible-base/roles/tails-config/templates/locale\" failed: No such file or directory (2)\nrsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1333) [sender=3.2.3]\n", "rc": 23}
```

Reply:
Clarified out of band:  This can be bootstrapped from <https://github.com/freedomofpress/securedrop/pull/6712#issuecomment-1569440679>, skipping step (3), and committing those changes onto *this* branch rather than the localization-specific testing branch I suggested there.

Reply:
Confirmed, fresh install working with change above.